modules/platforms/cpp/core-test/include/ignite/binary_test_defs.h

```diff
@@ -76,12 +76,12 @@ namespace ignite_test
                 BinaryFields(int32_t val1, int32_t val2, int32_t rawVal1, int32_t rawVal2) :
                     val1(val1), val2(val2), rawVal1(rawVal1), rawVal2(rawVal2)
                 {
-                    // No-op.   
+                    // No-op.
                 }
 
                 friend bool operator==(const BinaryFields& one, const BinaryFields& two)
                 {
-                    return one.val1 == two.val1 && one.val2 == two.val2 && 
+                    return one.val1 == two.val1 && one.val2 == two.val2 &&
                         one.rawVal1 == two.rawVal1 &&one.rawVal2 == two.rawVal2;
```

Comment:
Please add a space after `&&` in `&&one.rawVal2 == two.rawVal2`

Reply:
Wow, that's what I call attentiveness )